app/views/planning_applications/assessment/local_policies/edit.html.erb

```diff
@@ -18,19 +18,19 @@
 
 <%= render "table" %>
 
-<div class="govuk-button-group">
-  <%= link_to "Back", planning_application_assessment_tasks_path(@planning_application), class: "govuk-button govuk-button--secondary" %>
-  <%= link_to "Add new consideration", new_planning_application_assessment_local_policy_area_path(@planning_application), class: "govuk-link govuk-body", class: "govuk-button govuk-button--primary" %>
-</div>
+<%= link_to "+ Add new consideration", new_planning_application_assessment_local_policy_area_path(@planning_application), class: "govuk-link govuk-body" %>
```

Comment:
Is the `+` in this string intentional?

Reply:
Yes - new global style for adding elements to tables https://www.figma.com/file/AXJ7hvBaXVyJnh74Grwbml/BoPS-Prototype?type=design&node-id=9338-4000&mode=design&t=r5jxryDnwhdDvXIO-4